File: ui/polaris-gateway/Domain/DocumentExtraction/CaseDocument.cs
using System;
using Newtonsoft.Json;

namespace RumpoleGateway.Domain.DocumentExtraction
{
	public class CaseDocument
	{
		[JsonProperty("documentId")]
		public string DocumentId { get; set; }

		[JsonProperty("fileName")]
		public string FileName { get; set; }

		[JsonProperty("createdDate")]
		public DateTime CreatedDate { get; set; }

		[JsonProperty("cmsDocType")]
		public CmsDocType CmsDocType { get; set; }
	}
}

File: polaris-ui/src/app/features/cases/domain/CaseDocument.ts
import { CmsDocCategory } from "./CmsDocCategory";
import { CmsDocType } from "./CmsDocType";

export type CaseDocument = {
  documentId: number;
  fileName: string;
  createdDate: string;
  cmsDocCategory: CmsDocCategory;
  // documents in CMS are not guaranteed to have a cmsDocType
  cmsDocType: CmsDocType;
};
